code/parachain/frame/composable-maths/src/dex/constant_product.rs

```diff
@@ -86,6 +86,59 @@ where
 	Ok(ao)
 }
 
+/// Compute the amount of the output token given the amount of the input token.
+///
+/// Returns a tuple containing `(a_out, fee)`.
+/// To get `a_out` without accounting for the fee, set `f = 0`.
+///
+/// **NOTE:** Weights must be normalized.
+///
+/// From https://balancer.fi/whitepaper.pdf, equation (15)
```

Comment:
Let's refer our own doc as that has the fee included (while also having this reference) and we keep it up to date

Reply:
c932d03d98fc89a809c7d0c6fa8839067e2a06f8